renovate.json

```diff
@@ -4,6 +4,7 @@
   ],
   "dependencyDashboardApproval": true,
   "labels": ["PR: Dependency ⬆️"],
```

Comment:
Nit: we can also add the automation label.

Reply:
```suggestion
  "labels": ["PR: Dependency ⬆️", "repo automation 🤖" ],
```